tests/test_abstract_datamodel.py

```diff
@@ -9,7 +9,7 @@
 
 def test_roman_datamodel():
     roman_datamodels = pytest.importorskip("roman_datamodels.datamodels")
-    import roman_datamodels.tests.util as rutil
+    rutil = pytest.importorskip("roman_datamodels.tests.util")
```

Comment:
I no longer see the `roman_datamodels.tests.util` submodule in roman_datamodels. `mk_level2_image` is available from `roman_datamodels.maker_utils`.
```suggestion
    rutil = pytest.importorskip("roman_datamodels.maker_utils")
```
What options are there for updating the stpipe CI to run this test? At the moment I'm failing to find a run that installs roman_datamodels and runs the stpipe tests.

Reply:
There should be a `datamodels` install target which installs `stdatamodels` and `roman_datamodels`. One of the test cases can specify that additional target.

Reply:
Why `roman_datamodels` is not mentioned in test dependencies:
https://github.com/spacetelescope/stpipe/blob/5369e85b35c99280cd578b05c50f40f3ccb70d7f/pyproject.toml#L22-L35
And not in install dependencies:
https://github.com/spacetelescope/stpipe/blob/5369e85b35c99280cd578b05c50f40f3ccb70d7f/pyproject.toml#L14-L19